### ISSUE TITLE
Emit warning when the input `.toml` file overwrites the cell configuration in a `.pdb` file 

### ISSUE THREAD

**Issue by antoinewdg** [lumol-org/lumol#115]
Right now this happens completely silently.
Implement something like there https://github.com/lumol-org/lumol/blob/8ee3657495bf479acbfb5fc0644087ba630c0e98/src/input/src/simulations/system.rs#L54-L58.

**Comment by Luthaf** [comment]
This need to work not only for PDB files, but for any file with an unit cell: NetCDF, TNG, ... 

In order to do this, we can check if the system has a non-infinite unit cell before setting the cell from the input, and send a warning in this case.

**Comment by antoinewdg** [comment]
Do we have a way to test that input files generate a warning in the tests of the `input` crate ?

**Comment by Luthaf** [comment]
No, not yet. it should be easy to add to the test framework by using a custom logger and checking the messages at warning level.
